exatomic/widgets/traits.py

```diff
@@ -118,11 +118,17 @@ def tensor_traits(uni):
     except: idxs = [list(grp.index) for i, grp in grps]
     return {'tensor_d': grps.apply(lambda x: x.T.to_dict()).to_dict(), 'tensor_i': idxs}
 
+def freq_traits(uni):
+    grps = uni.frequency.groupby('frequency')
+    try: idxs = list(grps.groups.keys())
+    except: idxs = [list(grp.index) for i, grp in grps]
+    #print(idxs)
+    return {'freq_d': grps.apply(lambda x: x.T.to_dict()).to_dict(), 'freq_i': idxs}
```

Comment:
Curious about the intentions for frequency traits.  Perhaps it might make a good issue to discuss the current state of frequency data and viz.

Reply:
It was something I was working on before that I thought would be interesting to implement I have just commented out all of the code I made for it since it doesn't work right now. Now with working on this vibrational averaging code I think it may be good to look more into it in the near future.

Reply:
Yeah it's likely the existing frequency code is among the most un-tested code in the library.  I definitely recommend continuing to vet what's there.

Reply:
Yeah, I'm just trying some things out. Will probably open up an issue in the coming months so I have time to make some concise ideas on what we could do with it.